users/stanrc85/stanrc85.c

```diff
@@ -56,26 +56,15 @@ bool process_record_user(uint16_t keycode, keyrecord_t *record) {
     if (!record->event.pressed) {
       uint8_t mods = get_mods();
       clear_mods();
-      send_string_with_delay_P(PSTR("make " QMK_KEYBOARD ":" QMK_KEYMAP), 10);
+      send_string_with_delay_P(PSTR("qmk compile -kb " QMK_KEYBOARD " -km " QMK_KEYMAP), 10); //New way
+      send_string(SS_TAP(X_ENTER));
```

Comment:
`\n` works just as well here, and probably saves 2 bytes :)

Reply:
Save all the bytes!